examples/http.rs

```diff
@@ -1,9 +1,9 @@
 extern crate cookie_factory;
 
-use std::{str, iter::repeat};
+use std::{str};
```

Comment:
braces look weird when there's only one import

Reply:
We might want to just inline the std::str where it's used

Reply:
Done